cmd/integration_test/build/buildspecs/snow-test-eks-a-cli.yml

```diff
@@ -2,8 +2,8 @@ version: 0.2
 
 env:
   variables:
-    INTEGRATION_TEST_MAX_EC2_COUNT: 180
```

Comment:
Will this count suffice?

Reply:
We have limited compute resource for only 3 snow devices, I will increase the number gradually in the subsequent PRs.